<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Result Sheet</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>551</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_teacher">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Students</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addStudentButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>470</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add new Student</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addGroupButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add new Group</string>
      </property>
     </widget>
     <widget class="QComboBox" name="groupStudents">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>321</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addSubjectButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>470</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add new Subject</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showSubjects">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>150</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Subjects</string>
      </property>
     </widget>
     <widget class="QComboBox" name="teacherGroups">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>321</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="showStudents">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>110</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Students</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Groups</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="resultList">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>321</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="teachersList">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>321</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>teacher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showGroups">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>70</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Groups</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>-10</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Result Sheet</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
     <widget class="QPushButton" name="removeGroup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Group</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeStudent">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>440</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Student</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeSubject">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>440</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Subject</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>login</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>146</x>
        <y>190</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="enterToApp">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>270</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>enter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="loginEntry">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>104</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordEntry">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>170</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QComboBox" name="teacherListForAdmin">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>301</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Teachers</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addNewTeacher">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>450</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add new teacher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeTeacher">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Teacher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="teacherName">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>200</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="teacherAdmNo">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>270</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="teacherLogin">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>340</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="teacherPass">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>400</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Teacher full name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Teacher Reg No:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>350</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>410</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="adminClicked">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Login As Administrator</string>
      </property>
     </widget>
     <widget class="QPushButton" name="teacherClicked">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Login As Teacher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="studentClicked">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>220</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Continue As Student</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>585</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
